src/services/environment-manager.js

```diff
@@ -68,13 +72,41 @@ function EnvironmentManager(config) {
       to: toEnvironmentId,
     };
 
-    return agent
+    const deploymentRequestResponse = await agent
       .post(`${config.serverHost}/api/deployment-requests`)
       .set('Authorization', `Bearer ${authToken}`)
       .set('forest-origin', 'Lumber')
       .set('forest-project-id', config.projectId)
-      .send(DeploymentRequestSerializer.serialize(deploymentRequest))
-      .then(() => true);
+      .send(DeploymentRequestSerializer.serialize(deploymentRequest));
+
+    const jobId = deploymentRequestResponse.body.meta.job_id;
```

Comment:
error handling ?? If there is an error on the deployment request we should check it before doing the rest